README.md

```diff
@@ -1227,6 +1236,75 @@ address validation to work.
       license: "us-core-cloud" # This is the default license, but can be changed to any of the licenses listed here: https://smartystreets.com/docs/cloud/licensing
 ```
 
+### Validating an Address
+
+We have provided a `cfa:address` live template which provides form fields for Street Address, Apt
+Number,
+City, State and Zipcode. The address fragment in the live template has two parameters, `validate`
+and
+`inputName`. The `validate` parameter is a boolean (true or false) that determines whether the
+address will be
+validated by Smarty. The `inputName` parameter is the name of the associated address that will be
+linked
+to each of the Street Address, Apartment Number (as Street Address 2), City, State and Zipcode
+fields. In example, a inputName
+of
+`homeAddress` will result in the following fields:
+
+- `homeAddressStreetAddress1`
+- `homeAddressStreetAddress2`
+- `homeAddressCity`
+- `homeAddressState`
+- `homeAddressZipcode`
+
+**Note that you will need to include all the associated fields in your Flow Inputs Class file.
+Meaning
+if you give the address fragment an inputName of `homeAddress` you will need to include all the
+above-mentioned fields in your Flow Inputs Class file.**
+
+### Storage of Validated Addresses
+
+The validated address will be stored in the database in the `submissions` table's `input_data` JSON
+column.
+The JSON will be in the form of:
+
+```JSON
+{
+  "homeAddressStreetAddress1": "1719 5th Street",
+  "homeAddressStreetAddress2": "Apt D",
+  "homeAddressCity": "Berkeley",
+  "homeAddressState": "CA",
+  "homeAddressZipCode": "94710",
+  "_validatehomeAddress": "true",
+  "homeAddressStreetAddress1_validated": "1719 5th St Apt D",
+  "homeAddressStreetAddress2_validated": "D",
+  "homeAddressCity_validated": "Berkeley",
+  "homeAddressState_validated": "CA",
+  "homeAddressZipCode_validated": "94710-1738"
+}
+```
+
+Note that both the original address and the validated address are stored, with the validated fields
+having the suffix `_validated`.
+
+Also note the input prefixed with `_validate` which appears here as `validatehomeAddress` with a
+value of true.
+This is a hidden field which is applied within the address fragment. This field is used to determine
+whether Smarty validation for that address should be performed. The before mentioned `validate`
+parameter
+of the address fragment is used to determine the value of this hidden field.
+
+### Viewing the validated address
+
+We have provided a live template `cfa:addressValidationScreen` which will display a radio input with
+selections for the validated address that came back from Smarty if one was found, and the original
+address the user entered for them to select between. When using this live template you will be asked
+to enter values for the screen title, content replacement placeholder (this is a Thymeleaf variable
```

Comment:
The parameters might be better defined in a list format here.  Might be easier to read.

Reply:
A table mayhaps?


 Parameter           | Default | Type                              | Description                                                       |
| ------------------- | ------- | --------------------------------- | ----------------------------------------------------------------- |
| title               | n/a     | String                            | A title for the whole screen.                                     |
| content             | n/a     | Thymeleaf identifier              | An identifier that matches the `ref`.                             |
| ref                 | n/a     | String                            | A unique identifier for the form content. Should match `content`. |
| addressInputToCheck | n/a     | Name of the input being verified. |                                                                   |
| inputName           | n/a     | String                            | Input name of the radio input that will show on screen.           |
| editAddressURL      | n/a     | String                            | Location to go back to edit the previously entered address.       |

Reply:
Went with the table :)